samples/nextjs/src/components/GraphQL-ConnectedDemo/index.tsx

```diff
@@ -98,9 +99,9 @@ const GraphQLConnectedDemo = (props: StyleguideComponentProps): JSX.Element => {
 
 /**
  * Will be called during SSG
- * @param rendering
- * @param layoutData
- * @param context
+ * @param rendering {ComponentRendering}
+ * @param layoutData {LayoutServiceData | null}
```

Comment:
@ambrauer Probably, it can't be null? Should we change in `ComponentPropsFetchFunction` (in sitecore-jss-nextjs/component-props.ts) layoutData to be only LayoutServiceData? I see that you removed null option

Reply:
@sc-illiakovalenko You're right, that's no longer nullable. Fixed.